### PR TITLE
style-guide: add emphasis section

### LINKED ISSUE #11964: STYLE-GUIDE: document use or non-use of bold and italics
From what I've seen, bold and italics are not allowed in tldr pages to not conflict with client's rendering of placeholders, so it would be nice to document this decision and the reasoning.

### PR DESCRIPTION
**Pull request by vitorhcl**
Closes #11964.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: contributing-guides/style-guide.md [hunk 1]
```diff
@@ -330,6 +330,11 @@ For instance, instead of `Listing all files:`, `List all files:` can be used as
  `ls`
 ```
 
+## Emphasis
+
+- Don't use *itallics* and **bold** in the pages. These are reserved for client emphasis of placeholders.
+- Don't uppercase words unnecessarily. These are reserved to special cases like acronyms, brand names and notes on a description.
```
Comment (gutjuri) [inline contributing-guides/style-guide.md:336]:
```suggestion
- Do not use *italics* and **boldface** in the pages. These are reserved for client emphasis of placeholders.
- Do not uppercase words unnecessarily. These are reserved to special cases like acronyms, brand names and notes on a description.
```

**Review by gutjuri — COMMENTED** [review]
Thanks. I'd use "do not" instead of "don't" because it is more authoritative. Also, the current style guide uses "do not" more often that "don't" (6 times vs 1 time).

**Review by kbdharun — APPROVED** [review]
LGTM, thanks for the addition.

File: contributing-guides/style-guide.md [hunk 2]
```diff
@@ -330,6 +330,11 @@ For instance, instead of `Listing all files:`, `List all files:` can be used as
  `ls`
 ```
 
+## Emphasis
+
+- Do not use *italics* and **boldface** in the pages. These are reserved for client emphasis of placeholders.
+- Do not uppercase words unnecessarily. These are reserved to special cases like acronyms, brand names and notes on a description.
```
Comment (gutjuri) [inline contributing-guides/style-guide.md:336]:
```suggestion
- Do not use ALL CAPS unnecessarily. These are reserved for special cases like acronyms, brand names and notes on a description.
```

Comment (kbdharun) [reply contributing-guides/style-guide.md:336]: Maybe we can rephrase it to something like "do not capitalize words in middle of sentences unnecessarily ...". I don't think we use all caps in non-acronym instances in pages.

Comment (gutjuri) [reply contributing-guides/style-guide.md:336]: > Maybe we can rephrase it to something like "do not capitalize words in middle of sentences unnecessarily ...". I don't think we use all caps in non-acronym instances in pages.

Are we talking about instances of words written in ALL CAPS or words that are Unecessarily Capitalised Only At The First Character?

Comment (kbdharun) [reply contributing-guides/style-guide.md:336]: > Are we talking about instances of words written in ALL CAPS or words that are Unecessarily Capitalized Only At The First Character? This should be clarified in the proposed change.

Exactly, we are talking about the latter (words that are unnecessarily capitalized) here.

```suggestion
- Do not capitalize words in descriptions unnecessarily. We allow capitalization of starting words and special cases like acronyms, brand names and notes.
```

Comment (vitorhcl) [reply contributing-guides/style-guide.md:336]: > Exactly, we are talking about the latter (words that are unnecessarily capitalized) here.

Well, I was talking about words writer in ALL CAPS :sweat_smile:, I thought it's more common for people to do this for emphasis. 

Since we had thought of different things, I think it would be nice to document both of them.

Comment (gutjuri) [reply contributing-guides/style-guide.md:336]: > > Are we talking about instances of words written in ALL CAPS or words that are Unecessarily Capitalized Only At The First Character? This should be clarified in the proposed change.
> 
> Exactly, we are talking about the latter (words that are unnecessarily capitalized) here.

I don't think that this should be covered in the style guide since it is already covered by the usual English grammar rules.

Comment (kbdharun) [reply contributing-guides/style-guide.md:336]: I agree with @gutjuri here, we could drop this point and we can change the above point (with -) to a normal paragraph.

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
